qa-tests-backend/codenarc-rules.groovy

```diff
@@ -110,7 +110,7 @@ ruleset {
     AbstractClassWithoutAbstractMethod 
     AssignmentToStaticFieldFromInstanceMethod 
     BooleanMethodReturnsNull 
-    BuilderMethodWithSideEffects
+    // BuilderMethodWithSideEffects
```

Comment:
This is disabled as some creation methods in Kubernetes class returns void.

Reply:
Please put this comment inside the file.

Reply:
I added `@SuppressWarnings('BuilderMethodWithSideEffects')` on methods so we can slowly convert them to return something